scipy/stats/_qmc.py

```diff
@@ -1223,6 +1223,8 @@ class Sobol(QMCEngine):
     bits : int, optional
         Number of bits of the generator. Control the maximum number of points
         that can be generated, which is ``2**bits``. Maximal value is 64.
+        It also conditions the return type to be either 32 or 64 bits arrays.
```

Comment:
```suggestion
        It also conditions the return dtype to be either ``np.float32`` or ``np.float64``.
```
(If that is correct)